pennylane_lightning/core/simulators/lightning_qubit/bindings/LQubitBindings_nb.hpp

```diff
@@ -96,7 +139,8 @@ void registerBackendSpecificStateVectorMethods(PyClass &pyclass) {
 
     pyclass.def(
         "getState",
-        [](const StateVectorT &sv, nb::ndarray<ComplexT, nb::numpy> &state) {
+        [](const StateVectorT &sv,
+           nb::ndarray<ComplexT, nb::numpy, nb::c_contig> &state) {
```

Comment:
```suggestion
           nb::ndarray<ComplexT, nb::c_contig> &state) {
```

Any reason this shouldn't support all array types?